vagrant/scripts/build_rust_rpms.sh

```diff
@@ -0,0 +1,7 @@
+yum install -y openssl-devel
+yum install -y cargo rpm-build
+
+[ -f /root/.cargo/bin/sccache ] && echo "sccache already installed. Skipping." || cargo install sccache
+
+cd /integrated-manager-for-lustre \
+    && CARGO_HOME="/root/.cargo" SCCACHE_CACHE_SIZE="40G"  SCCACHE_DIR="/root/.cache" RUSTC_WRAPPER="/root/.cargo/bin/sccache" make copr-rpms
```

Comment:
We should probably set these in the profile instead of putting them in the env each time. That way user's can run the make command directly.